src/OpenSage.Game/Input/Cursors/CursorManager.cs

```diff
@@ -35,7 +35,13 @@ public CursorManager(AssetStore assetStore, ContentManager contentManager, GameW
 
         public void SetCursor(string cursorName, in TimeInterval time)
         {
-            if (!_cachedCursors.TryGetValue(cursorName, out var cursor))
+            SetCursor(cursorName, 0, time);
+        }
+
+        public void SetCursor(string cursorName, uint direction, in TimeInterval time)
```

Comment:
Could direction stay an enum here? I think the casting to an integer should happen inside this method

Reply:
Theoretically, there may be cursors that have other directions layout, like having four directions. So think the casting is necessary here.